popup.js

```diff
@@ -1,9 +1,57 @@
 function getHighlightedText(callback) {
-  chrome.tabs.executeScript( {
-    code: "window.getSelection().toString();"
+  // chrome.tabs.executeScript({
```

Comment:
Yeah, it's an old implementation. In fact, getHighlightedText has been replaced by getHighlightedObject which is superior b/c it returns the whole object...which can then be used to extract the highlighted div position and also the text separately in two different funcitons.